src/test/test-models/good/function-signatures/distributions/univariate/continuous/std_normal/std_normal_log.stan

```diff
@@ -0,0 +1,41 @@
+data { 
+  int d_int;
+  int d_int_array[d_int];
+  real d_real;
+  real d_real_array[d_int];
+  vector[d_int] d_vector;
+  row_vector[d_int] d_row_vector;
+}
+transformed data {
+  real transformed_data_real;
+
+  transformed_data_real <- std_normal_log(d_int);
+  transformed_data_real <- std_normal_log(d_real);
+  transformed_data_real <- std_normal_log(d_real_array);
+  transformed_data_real <- std_normal_log(d_vector);
+  transformed_data_real <- std_normal_log(d_row_vector);
+}
+parameters {
+  real p_real;
+  real p_real_array[d_int];
+  vector[d_int] p_vector;
+  row_vector[d_int] p_row_vector;
+
+  real y_p;
+}
+transformed parameters {
+  real transformed_param_real;
+
+  transformed_param_real <- std_normal_log(d_int);
+  transformed_param_real <- std_normal_log(d_real);
+  transformed_param_real <- std_normal_log(d_real_array);
+  transformed_param_real <- std_normal_log(d_vector);
+  transformed_param_real <- std_normal_log(d_row_vector);
+  transformed_param_real <- std_normal_log(p_real);
+  transformed_param_real <- std_normal_log(p_real_array);
+  transformed_param_real <- std_normal_log(p_vector);
+  transformed_param_real <- std_normal_log(p_row_vector);
+}
+model {  
+  y_p ~ normal(0,1); // in case anyone tries to run it
```

Comment:
Sorry for the delayed response; the last few weeks have been busy. I was looking at `student_t_log1` and there are no sampling statements:

https://github.com/stan-dev/stan/blob/ab66876b97e27e2d93ee0619fd5bd32293160cbd/src/test/test-models/good/function-signatures/distributions/univariate/continuous/student_t/student_t_log_1.stan#L1815-L1817

I agree that it's weird, but when I looked on the wiki or did a code search, I couldn't find where we test the sampling statements. Do you have any suggestions where else I can look?

Reply:
Is this what you're referring to, Bob? Otherwise, I'm not sure what to look for because no other file outside of the `student_t` directory comes up using `grep -l -r -I "student_t" ./src/test/ | uniq | less`.

https://github.com/stan-dev/stan/blob/972b4facd204d337287a538e2ff8627d1b1ae35e/src/test/unit/lang/parser/distribution_functions_test.cpp#L193-L196